packages/tools/merge-tree-client-replay/src/clientReplayTool.ts

```diff
@@ -342,6 +342,7 @@ export class ClientReplayTool {
                                         }
                                     }
                                 }
+                            // fallthrough
```

Comment:
optional nit: I'd rather see this fixed by putting `break;` in the previous case.

Reply:
Agreed, but I was nervous about changing the logic in some way. Adding the comment seemed safer (for me :)). Thanks for poking on this -- they could very well be bugs.